clients/src/main/java/org/apache/kafka/common/protocol/SecurityProtocol.java

```diff
@@ -26,6 +26,8 @@
     PLAINTEXT(0, "PLAINTEXT"),
     /** SSL channel */
     SSL(1, "SSL"),
+    /** SASL authenticated, non-encrypted channel */
+    PLAINTEXTSASL(2, "PLAINTEXTSASL"),
```

Comment:
OK, I can change my PR to use that instead. To check: is the proposed name better than what we have at the moment (SSLSASL and PLAINTEXTSASL).